src/main/java/io/github/railroad/config/Config.java

```diff
@@ -78,13 +92,21 @@ public void fromJson(JsonObject json) {
         }
     }
 
+    /**
+     * Copies the settings from the given config object
+     * @param config {@link Config}
+     */
     public void copyFrom(@Nullable Config config) {
         if (config == null)
             return;
 
         this.settings.get().copyFrom(config.getSettings());
     }
 
+    /**
+     * Gets the settings
+     * @return {@link Settings}
+     */
```

Comment:
Methods like this are completely self-explanatory. Javadoc here just bloats the file.